code/modules/1713/siege/piece.dm

```diff
@@ -26,8 +25,7 @@
 	var/nuclear = FALSE
 	var/reagent_payload = "none"
 	var/maxrange = 50
```

Comment:
Remove/rename this to `max_distance`, should not cause much trouble since it's almost the same for some reason